### PR TITLE
Adding entry for Quarkus

### PR DESCRIPTION
**Pull request by maxandersen**

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "024")
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala:6]: This needs to be the next highest number in the `ChangeLog` sequence.

File: src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    order = "001",
+    id = "001_add_quarkus_candidate",
+    author = "maxandersen"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "quarkus",
+      name = "Quarkus",
+      description =
+        "Quarkus is a Kubernetes Native Java framework tailored for OpenJDK HotSpot and GraalVM, crafted from best-of-breed Java libraries and standards.",
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala:19]: Is this just the CLI component? If so, can we make that explicit?

**Review by marc0der — COMMENTED** [review]
Two small comments, but apart from that LGTM.

File: src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala (hunk 1 above)
Comment (maxandersen) [reply src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala]: updated, set to 48.

File: src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala (hunk 2 above)
Comment (maxandersen) [reply src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala:19]: I was looking at micronaut for uniform messaging and it does not say only cli even though it seem to be just a cli.

I can change it but what todo with micronaut one ?

Comment (marc0der) [reply src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala:19]: It's not a biggie I guess. I noticed that Spring Boot also doesn't mention anything about CLI.

Comment (maxandersen) [reply src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala:19]: okey :) so we are good to go on this ?

File: src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala (hunk 1 above)
Comment (marc0der) [reply src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala]: Sorry, I think the next highest is 49.

**Comment by marc0der** [comment]
@maxandersen, I think this is good to go as soon as you bump the changelog to 49.

**Comment by maxandersen** [comment]
@marc0der done.

**Comment by marc0der** [comment]
Thanks @maxandersen,
It looks like the build is currently failing because Cuba's CDN is blocking URL validation. I'll recheck tomorrow morning, and if not resolved, I'll remove those migrations.

File: src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala (hunk 1 above)
Comment (marc0der) [reply src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala]: @maxandersen I had to fix a defunct migration on master, so your branch needs a rebase, sorry. Also, with all the incoming traffic, can you bump your changelog to 53, please. I feel like an air traffic controller :stuck_out_tongue:

**Comment by marc0der** [comment]
@maxandersen I had to fix a defunct migration on master, so your branch needs a rebase, sorry. Also, with all the incoming traffic, can you bump your changelog to 53, please. I feel like an air traffic controller :stuck_out_tongue:

**Comment by maxandersen** [comment]
rebased and use 53 in changelog.

**Review by marc0der — APPROVED** [review]

**Pull request merged**